tf_keras/tools/pip_package/setup.py

```diff
@@ -70,7 +70,6 @@
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
-        "Programming Language :: Python :: 3.12",
```

Comment:
Is this intended?

Reply:
Its not intended, but came from sync from cl/606736318 in commit https://github.com/keras-team/tf-keras/commit/7d027d62bf7ae5dc31d149e19a2bc2f45f1d81f5
There's some issue that `tf-keras` GitHub PR initiated changes are not going the other way. Have a todo on it to chat with you tomorrow.

Left it as-is so it aligns and also `pip_package/setup.py` is something we don't use.  We use the `oss_setup.py` and `pip_build.py`